pyogrio/tests/test_raw_io.py

```diff
@@ -775,6 +775,19 @@ def test_read_datetime_millisecond(test_datetime):
     assert field[1] == np.datetime64("2020-01-01 10:00:00.000")
 
 
+def test_read_datetime_tz(test_datetime_tz):
```

Comment:
```suggestion
def test_read_datetime_as_string(test_datetime_tz):
```

This is testing both the `datetime_as_string` parameter as well as ensuring that the string values include timezone; probably best to name the test case after the parameter.